Packs/FeedPlainText/Integrations/FeedPlainText/FeedPlainText_description.md

```diff
@@ -0,0 +1,94 @@
+## Plain Text Feed Integration
```

Comment:
please post screenshot of this from demisto

Reply:
posted